network-layer/Readme.md

```diff
@@ -0,0 +1,31 @@
+### Goal:
+1. Configure the network according to the scheme, using only static routing
+2. Verify the network is working with the ping command (all interfaces from everywhere must ping)
```

Comment:
Doesn't look like Markdown.

Reply:
I will check that out, in different readers looks different.

Reply:
ping